src/openfunction/runtime.ts

```diff
@@ -243,4 +261,216 @@ class DaprRuntime extends OpenFunctionRuntime {
 
     return Promise.allSettled(actions);
   }
+
+  /**
+   * Save the data to the state store
+   * @param data The data for save operation
+   * @param db The state store to save the data
+   * @returns The promise of the save action being executed.
+   */
+  #save(data: object, db?: string): Promise<object> {
```

Comment:
[suggestion] In case we would add Dapr Configuration / Secret Store components, let's use `state` as the prefix for these methods. :)